api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb

```diff
@@ -0,0 +1,31 @@
+module MnoEnterprise::Concerns::Controllers::Jpi::V1::Impac::TenantDashboardsController
```

Comment:
Shouldn't it be an admin controller? I thought this was used to load the admin dashboards in your new section in the admin panel?

Reply:
Yes true, this should be in the admin controller